spec/integration/data_stream/fields/fields.spec.yml

```diff
@@ -90,6 +90,7 @@ spec:
         - wildcard
         - version
         - unsigned_long
+        - counted_keyword
```

Comment:
We would need to add a patch to the `versions` at the end of the document, so this is not available in previous versions.